src/components/PeoplePicker/PeoplePicker.less

```diff
@@ -176,33 +180,51 @@
   margin: 0 0 10px 0;
```

Comment:
can this be changed to margin-bottom or is there a reason to keep the other coordinates? (same with the padding)